source-syntax/scribblings/source-syntax.scrbl

```diff
@@ -12,7 +12,8 @@
   Return a procedure that accepts a syntax object from @racket[expanded]
   and returns the outermost syntax object in @racket[orig] that has the same
   location as the given syntax object.
-
+  If no syntax object in @racket[orig] has the same location as the given
+  syntax object, the procedure repeats with parent of the given syntax object.
```

Comment:
Nit: "the parent"?
Otherwise the whole thing LGTM.

Reply:
oh wow, thank you